src/WebJobs.Script/Description/FunctionMetadata.cs

```diff
@@ -36,6 +36,8 @@ public FunctionMetadata()
 
         public ScriptType ScriptType { get; set; }
 
+        public string Language { get; set; }
```

Comment:
The enum above ScriptType already captures the language - it will be confusing to have two

Reply:
Tracks removing scriptType enum https://github.com/Azure/azure-functions-host/issues/3280

Reply:
I'd rather we just do this now rather than add a new duplicate property and incur debt and confusion. Shouldn't be that hard, right? Or perhaps just do that in a separate change and rebase this on that?